src/corelibs/U2Gui/src/options_panel/OPWidgetFactory.h

```diff
@@ -85,6 +89,7 @@ class U2GUI_EXPORT OPFactoryFilterVisitor : public OPFactoryFilterVisitorInterfa
     OPFactoryFilterVisitor(ObjectViewType _objectViewType, QList<DNAAlphabetType> _objectListAlphabet)
         : OPFactoryFilterVisitorInterface(), objectViewType(_objectViewType), objectAlphabetType(DNAAlphabet_RAW), objectAlphabets(_objectListAlphabet) {
     }
+    OPFactoryFilterVisitor(ObjectViewType _objectViewType, const QList<const DNAAlphabet *> &_objectListAlphabet);
```

Comment:
The method looks identical to the old one above.

Reply:
Removed, the unneeded changes